test/authenticator.spec.ts

```diff
@@ -333,35 +305,27 @@ describe('Decentraland Crypto', function () {
       let result = await Authenticator.validateSignature(
         'message',
         chain,
-        new HttpProvider(
-          'https://mainnet.infura.io/v3/640777fe168f4b0091c93726b4f0463a'
-        )
+        new HTTPProvider('https://rpc.decentraland.org/mainnet')
       )
 
-      expect(result.message).satisfies((message: string) =>
-        message.startsWith(
-          'ERROR. Link type: ECDSA_EPHEMERAL. Ephemeral key expired.'
-        )
+      expect(result.message).toMatch(
+        'ERROR. Link type: ECDSA_EPHEMERAL. Ephemeral key expired.'
       )
 
       // Since we are checking the ephemeral against 10 minutes ago, validation should pass
       result = await Authenticator.validateSignature(
         'message',
         chain,
-        new HttpProvider(
-          'https://mainnet.infura.io/v3/640777fe168f4b0091c93726b4f0463a'
-        ),
+        new HTTPProvider('https://rpc.decentraland.org/mainnet'),
```

Comment:
Can we use https://rpc.decentraland.org/mainnet?project=dcl-crypto-test so we can discriminate the usage?